src/app/hierarchical-grid/hierarchical-grid-column-pinning/hierarchical-grid-toolbar-pinning.component.ts

```diff
@@ -9,15 +9,16 @@ import { ActivatedRoute } from '@angular/router';
 })
 export class HGridToolbarPinningComponent implements OnInit {
     public localdata;
-    public dark = false;
+    public dark = "";
```

Comment:
I see why it is done this way (setting it as string) and it works out nicely when you receive a string value from the query params.

In general, IMO, it is good to keep the value boolean, and the check whether it is a "true" string being moved from the template to the place where you set the value on line 21:

```
this.dark = params.dark === 'true' ? true : false;
```

Reply:
> 

Null coalescing operator serves a more specific purpose, so when I think about it the Logical OR operator would serve us best here.